submitter/adaptors/terraform_adaptor.py

```diff
@@ -1155,5 +1171,3 @@ def _terraform_customise(self):
         self._terraform_exec(command)
         command = ["sh", "-c", "apk -e info py3-pip || apk add py3-pip && pip3 install PyJWT" + LOG_SUFFIX]
```

Comment:
Isn't this now handled with the PostStart hook in [micado-manifest.yml](https://github.com/micado-scale/ansible-micado/blob/develop/roles/micado_master/start/templates/micado/micado-manifest.yml.j2#L305) ?

Reply:
I think we can remove that terraform_customise function now. I forgot to remove it.